### PR TITLE
Transaction signing example and explanation

### PR DESCRIPTION
**Pull request by jonathancross**
For a while now, I've wanted to provide an actual transaction signing example for users.

Not 100% sure this is the best location, but probably a good start.
Maybe better as a separate file and / or added to https://doc.satoshilabs.com/trezor-apps/ ?
I have screenshots as well which would make for a more friendly version on that site.

Feedback is welcome!

### CODE REVIEW

**Comment by prusnak** [comment]
Let's create a `/docs` directory, move USAGE.rst and images there.

**Comment by jonathancross** [comment]
@prusnak Would you like that as part of this PR or as a followup after it is merged?

**Comment by prusnak** [comment]
This PR would be better.

File: .gitignore [hunk 1]
```diff
@@ -4,7 +4,6 @@ python_trezor.egg-info/
 trezor.egg-info/
 *.pyc
 *.bin
-*.png
```
Comment (jonathancross) [inline .gitignore]: Allow png files again now that we are using these for screenshots.

**Comment by jonathancross** [comment]
Okay, now updated.  :-)

**Comment by prusnak** [comment]
:+1:

**Comment by jonathancross** [comment]
Cheers!

**Pull request merged**